easycorp/easy-deploy-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+    "bundles": {
+        "EasyCorp\\Bundle\\EasyDeployBundle\\EasydeployBundle": ["dev"]
```

Comment:
And by the way, your issue may come from the fact that you wrote `EasydeployBundle` instead of `EasyDeployBundle`